ui/panels-plugin/src/plugins/empty-chart/EmptyChart.tsx

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 The Perses Authors
+// Copyright 2022 The Perses Authors
```

Comment:
Haha this must be out-of-date in a few places huh

Reply:
So. Many. Places. 😂